test/controllers/classrooms_controller_test.rb

```diff
@@ -166,9 +166,16 @@ def setup
     assert_redirected_to root_path
   end
 
-  test "teachers cannot edit classrooms" do
+  test "teachers can edit their own classrooms" do
```

Comment:
Why are we changing this?  I don't think that we want  to make this change in behavior.  I'll check with the stakeholders this week if they want teachers to be able to edit anything on the classroom but my guess is no.